hpctestlib/apps/cp2k/__init__.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.typecheck as typ
+
+__all__ = ["Cp2k"]
+
+
+class Cp2k(rfm.RunOnlyRegressionTest):
```

Comment:
I now think that all the base checks related to the apps should have the name of the scientific system and not just cp2k and so on. Also, I now think that because this is not a generic cp2k class, the file should also be renamed from `__init__` to the benchmark name (like in the amber case, where the equivalent to this file is named `nve`). I guess this applies to the rest of the apps too.